theano/tensor/tests/test_assert_op_gradient.py

```diff
@@ -0,0 +1,16 @@
+import numpy
```

Comment:
the tests should be in the file theano/tensor/tests/test_opt.py. We should not make 1 test file per op, this would create too much files.